.github/workflows/branch.yml

```diff
@@ -90,6 +90,7 @@ jobs:
             gs://$BUCKET_OPENFIN/branch/${{ steps.vars.outputs.BRANCH }}
 
       - name: Build (Finsemble)
+        if: ${{ steps.vars.outputs.ENVIRONMENT == 'dev' || steps.vars.outputs.ENVIRONMENT == 'uat' }}
```

Comment:
only build Finsemble for Dev, UAT and Prod .. and consider dropping it entirely in the new year .. if we pick it up again (unlikely), we would probably look at using the [io.Connect](https://docs.interop.io/desktop/getting-started/how-to/migrate-to-io-connect/index.html) (i.e. rebadged Glue 42) product which has superseded Finsemble now (there is a migration path, I am told).